### PR TITLE
Prepare r1.1 (first pre-release as of July 4th)

### LINKED ISSUE #97: Create the release r1.1 of the RM documentation
**Problem description**
The RM documentation in GitHub should also be under release management. 

**Expected action**

- Create a r1.1 now based on issue #35 with the last larger change of the repository ("Update public release name") which was merged on July 4th (shortly before the release candidates of Commonalities and ICM). That could be candidate for r1.1 (if using a version it would be something as 0.1.0-rc) but as we haven't give the artefacts a version we might not need that).

**Additional context**
All changes in GitHub need to be reflected also in the RM Wiki pages to remain in synch.

### PR DESCRIPTION
**Pull request by hdamker**
#### What type of PR is this?

Add one of the following kinds:

* subproject management

#### What this PR does / why we need it:

Creating CHANGELOG.md and VERSION.md on branch temp-r1.1
Updating the past README on branch temp-r1.1

#### Which issue(s) this PR fixes:

Fixes #97 

#### Special notes for reviewers:

temp-r1.1 will afterwards also merged into main (some merge conflicts with the README.md need then to be resolved), as a base for r1.2 release information.

### CODE REVIEW

File: CHANGELOG
Comment (tanjadegroot) [inline CHANGELOG:1]: this file should have the .md extension

File: VERSION.md
Comment (tanjadegroot) [inline VERSION.md:1]: I though you did not want to create a version file for the r1.1, but OK with me. 
- Let's changes this into a .yaml file
- remove the "v" from the version number

File: README.md
Comment (tanjadegroot) [inline README.md:1]: we could remove the link to the fall24 release, or replace it in the 1.2 by a link to the page with all the meta-releases for ease of maintenance ? link = https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/14574702/CAMARA+meta-releases

File: CHANGELOG [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+# r1.1
+
+## Release Notes
+
+**This is the first pre-release of the Release Management documents with version 0.1.0.** 
+
+Content is based on the work done by the [Release Management Working Group](https://lf-camaraproject.atlassian.net/l/cp/scZC1AYy) and the results are located in the [Release Management repository](https://github.com/camaraproject/ReleaseManagement).
+
+**It provides the initial version of de guidelines describing how to releasing an API version, as well as the API-Readiness-Checklist template in which to record the availability of the artefacts required for the API release.**
```
Comment (tanjadegroot) [inline CHANGELOG:15]:
```suggestion
**It provides the initial version of de guidelines describing how to release an API version, as well as the API-Readiness-Checklist template in which to record the availability of the artefacts required for the API release.**
```

File: CHANGELOG
Comment (hdamker) [reply CHANGELOG:1]: done in https://github.com/camaraproject/ReleaseManagement/pull/106/commits/c331a2192c1ba6008ad25496883b35d1e44fe6b3

File: VERSION.md
Comment (hdamker) [reply VERSION.md:1]: done in https://github.com/camaraproject/ReleaseManagement/pull/106/commits/c331a2192c1ba6008ad25496883b35d1e44fe6b3

**Pull request merged**